tests/codeception/acceptance/SubThemeCest.php

```diff
@@ -101,7 +101,7 @@ public function testSubTheme(AcceptanceTester $I) {
   protected function runConfigImport(AcceptanceTester $I, $disable_config_ignore = FALSE) {
     $drush_response = $I->runDrush('pm-list --filter=name=stanford_ssp --format=json');
     $drush_response = json_decode($drush_response, TRUE);
-    $saml_enabled = $drush_response['stanford_ssp']['status'] == 'Enabled';
+    $saml_enabled = ($drush_response['stanford_ssp']['status'] == 'Enabled');
```

Comment:
why the `()`? what about just a triple equal?

Reply:
Just ensuring we capture the boolean.